all-in-one-apim/.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# See: https://help.github.com/articles/about-codeowners/
+
+# These owners will be the default owners for everything in the repo.
```

Comment:
These file shoud move to root folder and removed from the component folders. check other default files as well

Reply:
Fixed with https://github.com/wso2/product-apim/pull/13650/commits/af45e8ced0fc3b1441ec852b1d8c93d963a7d3b6